utils/model_uploader/trigger_ml_models_release.sh

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+# This program is run by "Model Auto-tracing & Uploading" 
+# & "Model Listing Uploading" workflow (See model_uploader.yml 
+# & model_listing_uploader.yml) to trigger ml-models-release 
+# Jenkins workflow.
+
+JENKINS_TRIGGER_TOKEN=$1
```

Comment:
How did we test this?

Reply:
I tested in on my repo using jenkins Sayali set up for me for testing already. And I talked to Sayali about testing it with actual credentials, and she recommends to have this PR merged first and then we can schedule a meeting to test it.